### DISCUSSION TITLE
Add Tweak For Disable Enable Music Theme, Or a Battery saver mode features

### DISCUSSION THREAD

**Discussion by Rahhide** [OnionUI/Onion#1149] [Ideas]
I think the background theme is annoying in my office , this features maybe saves the battery over all. or change all the background
theme music to stock haptic sound . is good experience for onion ui.
the battery saver can mod or lock the screen brightness or disable theme music (change to haptic sound or mute), disable wifi,
Thanks before. Love the Miyoo mini ❤️❤️❤️

**Comment by Phanturian** [comment]
The feature to toggle BGM on and off is a good idea. For now you can always go into the theme folder on the root of your SD card and delete bgm.mp3 in the sound folder. Maybe back it up first. The pic is just an example, as the CyberOnion theme with music is one of my favorites.
![example](https://github.com/OnionUI/Onion/assets/135469050/9370b4ee-dfe6-4068-ab08-39f31c264810)

**Reply by Rahhide** [reply]
Thanks for recomendation trick, i hope the next update lot of new features of tweaking.
